### PR TITLE
W-14322169-rtfctl-audit-logs-kt

### PR DESCRIPTION
**Pull request by kevintroller**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: modules/ROOT/pages/install-rtfctl.adoc [hunk 1]
```diff
@@ -556,6 +556,25 @@ Replicas:          1 desired | 1 updated | 1 total | 0 available | 1 unavailable
   hello-world-77f5897949-z54gb   Pending
 ----
 
+== Auditing rtfctl Operations
+
+The `rtfctl` command line tool (from vX.X.X and above) audits the `rtfctl` operations on the cluster. Helping cluster administrators and you to track operations that impact Runtime Fabric core software.
```
Comment (Crispy-Salesforce) [inline modules/ROOT/pages/install-rtfctl.adoc:561]: It seems that a connector is missing.
The sentence may read:
"The `rtfctl` command line tool (from vX.X.X and above) audits the `rtfctl` operations on the cluster and it helps cluster administrators and you to track operations that impact Runtime Fabric core software."

Comment (Crispy-Salesforce) [inline modules/ROOT/pages/install-rtfctl.adoc]:
```suggestion
The `rtfctl` command line tool (from vX.X.X and above) audits the `rtfctl` operations on the cluster and it helps cluster administrators and you to track operations that impact Runtime Fabric core software.
```

Comment (Crispy-Salesforce) [reply modules/ROOT/pages/install-rtfctl.adoc]: Would it make sense to say "cluster administrators" and "you"? What if "you" is also a cluster administrator?

**Comment by hegderohit89** [comment]
@SparshAgarwal can you provide some inputs to these docs please.

File: modules/ROOT/pages/install-rtfctl.adoc (hunk 1 above)
Comment (hegderohit89) [reply modules/ROOT/pages/install-rtfctl.adoc]: Yes, though its mainly aimed for cluster admins but any user can use it. So `you` makes better fir

File: modules/ROOT/pages/install-rtfctl.adoc [hunk 2]
```diff
@@ -556,6 +556,25 @@ Replicas:          1 desired | 1 updated | 1 total | 0 available | 1 unavailable
...
+
+* Audit logs are stored in a Kubernetes secret in the `rtfctl-audit` namespace.
+* Audit logs are cluster scoped and not attached to Runtime Fabric core software lifecycle.
+* Command executions continue to function if the execution fails to audit the operation.
+
+To audit the operations, run the following command:
+----
+$ sudo ./rtfctl audit-logs
+<add a sample output>
+----
```
Comment (kevintroller) [inline modules/ROOT/pages/install-rtfctl.adoc]: We need a full example:

- The command to run
- An example of running the command
- The output example that returns

File: modules/ROOT/pages/install-rtfctl.adoc (hunk 1 above)
Comment (kevintroller) [inline modules/ROOT/pages/install-rtfctl.adoc]: Also, we need to specify the version that you refer here from `vX.X.X`

File: modules/ROOT/pages/install-rtfctl.adoc (hunk 2 above)
Comment (hegderohit89) [reply modules/ROOT/pages/install-rtfctl.adoc]: command to run: 
`$ sudo ./rtfctl audit-logs`

output example that returns:
```
[{"timestamp":"2023-10-26 00:52:24.446 +0000 UTC","kubeContext":"test-k8scontext","command":["./rtfctl","restart","test-application","--app-namespace","8456f9c7-2149-4983-b02b-6b18ddd28efe","--namespace","rtf"]}]
```

File: modules/ROOT/pages/install-rtfctl.adoc [hunk 3]
```diff
@@ -556,6 +556,25 @@ Replicas:          1 desired | 1 updated | 1 total | 0 available | 1 unavailable
...
 
+== Auditing rtfctl Operations
+
+The `rtfctl` command line tool (from vX.X.X and above) audits the `rtfctl` operations on the cluster and it helps cluster administrators and you to track operations that impact Runtime Fabric core software.
+
+Consider the following points:
+
+* Audit logs are stored in a Kubernetes secret in the `rtfctl-audit` namespace.
+* Audit logs are cluster scoped and not attached to Runtime Fabric core software lifecycle.
+* Command executions continue to function if the execution fails to audit the operation.
```
Comment (hegderohit89) [inline modules/ROOT/pages/install-rtfctl.adoc]: Command executions remain operational even if the auditing process fails to audit the operation

**Review by dmerlob — DISMISSED** [review]

**Review by Crispy-Salesforce — APPROVED** [review]

**Pull request merged**